test/unittests/mongoBackend/mongoUpdateContext_test.cpp

```diff
@@ -5728,7 +5728,7 @@ TEST(mongoUpdateContextRequest, updateAttrWithId)
 *
 * updateAttrWithAndWithoutId -
 */
-TEST(mongoUpdateContextRequest, updateAttrWithAndWithoutId)
```

Comment:
Or defer this for the future with a "FIXME P10" with "We don't know what the hell is happening here but enabling this crashes the unittest binary. Sad but true :)"

;)